scripts/start-rpc-servers.go

```diff
@@ -102,6 +102,13 @@ func main() {
 				running = append(running, anvilCmd)
 			}
 
+			dataFolder := "./data/nitro-service"
```

Comment:
Where is the corresponding line where we specify that data should be stored in this folder? Is it baked in to `go-nitro`?

Reply:
It's part of the internal [RunRPCServer](https://github.com/statechannels/go-nitro/blob/8d7a7c11e6b3e46a2861d0415511995cdfa6bdee/internal/rpc/rpc.go#L64)

Reply:
Probably. We should update our `go-nitro` executable to take in a flag to specify the store location (with some sane default). That way the start-rpc-servers script can decide on the data folder and make sure it's clean beforehand.

Reply:
It slightly expands the scope of the PR but I've done this in https://github.com/statechannels/go-nitro/pull/1422/commits/96264f2e4aa2e958765b7c3614cccbff28fc443d go-nitro now accepts a `durablestorefolder` argument and we generate a random temp folder when we run our start rpc server script.

Reply:
Thanks! Perhaps update the PR title before merging 👍